Buildings/Controls/OBC/ASHRAE/G36/AHUs/MultiZone/SetPoints/ReliefDamper.mo

```diff
@@ -0,0 +1,143 @@
+﻿within Buildings.Controls.OBC.ASHRAE.G36.AHUs.MultiZone.SetPoints;
+block ReliefDamper "Control of actuated relief  dampers without fans"
+  parameter Modelica.SIunits.Pressure buiPreSet=0.05*248.84
+    "Building static pressure setpoint"
+    annotation(Evaluate=true);
+  parameter Real kp(min=0, unit="1") = 0.5
+    "Gain factor"
+    annotation(Dialog(group="Relief damper P-control parameter"));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput uBuiPre(
+    final unit="Pa",
+    quantity="PressureDifference")
+    "Measured building static pressure"
+    annotation (Placement(transformation(extent={{-120,0},{-80,40}}),
+      iconTransformation(extent={{-120,50},{-100,70}})));
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput uSupFan
+    "Supply Fan Status, on or off"
+    annotation (Placement(transformation(extent={{-120,-40},{-80,0}}),
+      iconTransformation(extent={{-120,-70},{-100,-50}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput yRelDamPos(
+    min=0, max=1, unit="1")
+    "Relief damper position"
+    annotation (Placement(transformation(extent={{80,-10},{100,10}}),
+      iconTransformation(extent={{100,-10},{120,10}})));
+
+  Buildings.Controls.OBC.CDL.Continuous.Sources.Constant zerDam(
+    final k=0)
+    "Close damper when disabled"
+    annotation (Placement(transformation(extent={{-60,-50},{-40,-30}})));
+  Buildings.Controls.OBC.CDL.Continuous.Sources.Constant buiPreSetpoint(
+    final k=buiPreSet)
+    "Building pressure setpoint"
+    annotation (Placement(transformation(extent={{-60,30},{-40,50}})));
+  Buildings.Controls.OBC.CDL.Continuous.LimPID damPosController(
+    final yMax=1,
+    final yMin=0,
+    Td=0.1,
+    Nd=1,
+    Ti=300,
+    final k=kp,
+    controllerType=Buildings.Controls.OBC.CDL.Types.SimpleController.P)
+    "Contoller that outputs a signal based on the error between the measured "
+    annotation (Placement(transformation(extent={{-20,30},{0,50}})));
+  Buildings.Controls.OBC.CDL.Logical.Switch swi
+    "Check if relief damper should be activated"
+    annotation (Placement(transformation(extent={{40,-10},{60,10}})));
+
+equation
+  connect(buiPreSetpoint.y, damPosController.u_s)
+    annotation (Line(points={{-39,40},{-22,40}}, color={0,0,127}));
+  connect(uBuiPre, damPosController.u_m)
+    annotation (Line(points={{-100,20},{-10,20},{-10,28}},
+      color={0,0,127}));
+  connect(uSupFan, swi.u2)
+    annotation (Line(points={{-100,-20},{-32,-20},{-32,0},{38,0}},
+      color={255,0,255}));
+  connect(damPosController.y, swi.u1)
+    annotation (Line(points={{1,40},{20,40},{20,8},{38,8}},
+      color={0,0,127}));
+  connect(zerDam.y, swi.u3)
+    annotation (Line(points={{-39,-40},{20,-40},{20,-8},{38,-8}},
+      color={0,0,127}));
+  connect(swi.y, yRelDamPos)
+    annotation (Line(points={{61,0},{90,0}},  color={0,0,127}));
+
+annotation (
+  defaultComponentName = "reliefDamper_multiZone",
```

Comment:
use shorter words, such as `relDam` (it is clear that it is for multi zone when used in a model)